jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/main/java/org/jbpm/console/ng/ht/client/editors/taskdetails/TaskDetailsViewImpl.java

```diff
@@ -95,14 +95,14 @@
     @Inject
     private PlaceManager placeManager;
 
-    private String[] priorities = { "0 - High", "1", "2", "3", "4", "5 - Medium", "6", "7", "8", "9", "10 - Low" };
+    private String[] priorities = { "0 - " + constants.High(), "1", "2", "3", "4", "5 - " + constants.Medium(), "6", "7", "8", "9", "10 - " + constants.Medium() };
```

Comment:
I am not sure if this is what @manstis meant, but the last constant here should be `constants.Low()`.